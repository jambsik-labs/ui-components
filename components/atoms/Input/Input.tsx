/** @jsxImportSource theme-ui */
import { Input as ThemeUIInput, InputProps } from 'theme-ui';

export const Input = (props: InputProps) => {
    return <ThemeUIInput {...props} />;
};
