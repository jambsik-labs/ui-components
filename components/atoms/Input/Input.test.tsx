import React from 'react';
import { render } from '@testing-library/react';
import { Input } from './Input';

describe('Input test', () => {
    it('Should render component without errors', () => {
        const { container } = render(<Input />);

        expect(container).toMatchSnapshot();
    });
});
