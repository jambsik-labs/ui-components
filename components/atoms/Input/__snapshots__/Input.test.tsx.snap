// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input test Should render component without errors 1`] = `
<div>
  <input
    class="css-1y7h7x1"
  />
</div>
`;
